manifests/server/set_table_attribute.pp

```diff
@@ -0,0 +1,22 @@
+define postgresql::server::set_table_attribute (
+  String  $db,
+  String  $table_name,
+  String  $table_attribute,
+  String  $table_attribute_value,
+  String  $psql_user         = $postgresql::server::user,
+  String  $psql_group        = $postgresql::server::group,
+  Integer $port              = $postgresql::server::port,
+  Hash    $connect_settings  = $postgresql::server::default_connect_settings,
+  String  $psql_path         = $postgresql::server::psql_path
+) {
+  postgresql_psql { "Set ${table_attribute}=${table_attribute_value} for ${table_name}" :
```

Comment:
This would conflict when the resource is used for tables with the same name on different databases.  I would go for: `${db}:ALTER TABLE \"${table}\" SET \"${attribute}\""`.